.githooks/pre-commit

```diff
@@ -1,13 +1,32 @@
 #!/usr/bin/env bash
 set -e
 
+eslint_installed=false 
+
+# Check for eslint
+if ! which eslint; then
+  printf "Not auto linting, please install eslint globally (npm i -g eslint)"
+else 
+  eslint_installed=true
+fi
+
 # Format any *.tf files that were cached/staged
 files=$(git diff --cached --name-only)
 for f in $files
 do
   if [ -e "$f" ] && [[ $f == *.tf ]]; then
     printf "Pre-Commit Git Hook Running: 'terraform fmt' \n\n"
-    terraform fmt $f
-    git add $f
+    terraform fmt "$f"
+    git add "$f"
   fi
-done
+
+  if $eslint_installed && [ -e "$f" ] && [[ $f == *.js ]]; then    
+    eslint --fix "$f"
```

Comment:
I think we'd like to use the eslint binary that is installed in node_modules, rather than expect a global install. We could run `yarn run lint:fix:all` or something similar. There are some aliased commands in package.json that might be worth a look.